source/FreeRTOS_DNS_Networking.c

```diff
@@ -96,7 +96,7 @@
                               struct freertos_sockaddr * xAddress,
                               struct xDNSBuffer * pxDNSBuf )
     {
-        BaseType_t xReturn = pdFALSE;
+        uint32_t xReturn = pdFALSE;
```

Comment:
I think we should do it the other way around. Make the return type of the function as `BaseType_t`. This will make it more consistent with the library and a tiny bit optimized on smaller (16-bit) platforms where `BaseType_t` is 16-bits.